aws/chef-bootstrap.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+# Code-dot-org Chef bootstrap script.
+# One-liner to run:
+# aws s3 cp s3://cdo-dist/chef/bootstrap.sh - | sudo bash -s -- [options]
+#
+# Options:
```

Comment:
This script was intentionally written in POSIX-compatible shell (pretty confident; I need to update the shebang and test to make sure) to be as portable and future-proof as possible, because Ruby is not yet installed on the system (we use Chef to install a specific version of Ruby). Chef maintains their installation script in shell for the same reason. The intent is to keep this script as short and simple as possible, and should only contain anything that is not possible to do within Chef (basically just install Chef), so I don't expect the complexity to grow.

More than half of the script is either static file contents or comments, so I don't believe the complexity is really all that great (especially when compared to installing and maintaining a bootstrap-level Ruby installation solely for the purposes of writing this script in Ruby).